content/general/firehose.textile

```diff
@@ -4,14 +4,14 @@ section: general
 index: 26
 ---
 
-Our Firehose can stream your realtime data published within the Ably platform directly to another streaming or queueing service. For example, all messages published by any device on a channel could be immediately streamed to Amazon Kinesis allowing you to process this data in realtime. Firehose is offered as part of our "Ably Reactor service":https://www.ably.io/reactor, however it is only available to Enterprise customers.
+Our Firehose can stream your realtime data published within the Ably platform directly to another streaming or queueing service. For example, all messages published by any device on a channel could be immediately streamed to Amazon Kinesis allowing you to process this data in realtime. Firehose is offered as part of our "Ably Reactor service":https://www.ably.io/reactor, however it is only available to "Enterprise customers":https://www.ably.io/pricing/enterprise.
```

Comment:
> Firehose is offered as part of our "Ably Reactor service":https://www.ably.io/reactor, however it is only available to "Enterprise customers":https://www.ably.io/pricing/enterprise.

Grammatical nitpick: pls don't join two sentences with a comma; use  semicolon or say, for example,

Firehose is offered as part of our "Ably Reactor service":https://www.ably.io/reactor but is currently only available to "Enterprise customers":https://www.ably.io/pricing/enterprise.